runtime/onert/backend/gpu_cl/open_cl/OpenclWrapper.cc

```diff
@@ -108,23 +108,33 @@ absl::Status LoadOpenCL()
 #endif
 }
 
+void UnloadOpenCL(void **libopencl)
+{
+  if (*libopencl)
+  {
+    dlclose(*libopencl);
+  }
+}
+
 #ifdef __WINDOWS__
 void LoadOpenCLFunctions(HMODULE libopencl)
 {
 #else
-void LoadOpenCLFunctions(void *libopencl, bool use_wrapper)
-{
 #ifdef __ANDROID__
+void LoadOpenCLFunctions(void **libopencl, bool use_wrapper)
```

Comment:
maybe leave ANDROID code  as-is...

Reply:
I modified this because "use_wrapper" is not used when not on Android.